addons/sms/models/sms_template.py

```diff
@@ -23,7 +23,9 @@ def default_get(self, fields):
         help="The type of document this template can be used with")
     model = fields.Char('Related Document Model', related='model_id.model', index=True, store=True, readonly=True)
     body = fields.Char('Body', translate=True, required=True)
-    lang = fields.Char('Language', placeholder="${object.partner_id.lang}")
+    ref_ir_act_window = fields.Many2one('ir.actions.act_window', 'Sidebar action', readonly=True, copy=False,
```

Comment:
I would name it sidebar_action_id instead of ref_ir_act_window that is not really easy to understand (copy pasting from mail I suppose ^^).